### PR TITLE
Add Navigation, Supreme page, and handleDelete updates

### PR DESCRIPTION
**Pull request by CTGitWhiz**
In this pull request, I have made several changes:

- Added a Navigation component that makes it easier for users to navigate between the Home and Supreme pages.
- Created a new Supreme page that showcases the best images for each category based on user ratings.
- Updated the handleDelete function in the ImageList component to remove associated range values when an image is deleted from the Home page.
- Improved accessibility for better user experience
- Fixed the 404 error in the console by adding the favicon.ico back to the public folder.

### CODE REVIEW

File: pages/components/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  &:hover {
+    color: #ffcc00;
+  }
+`;
+
+export default function Navigation() {
+  return (
+    <Nav>
+      <Link href="/" aria-label="PicJudge Home">
+        <NavItem>[Home]</NavItem>
```
Comment (pung-one) [inline pages/components/Navigation.js:41]: do you need the eckige klammern here?

File: pages/components/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      </Link>
+      <Link href="/supreme" aria-label="Supreme Page">
+        <NavItem>[Supreme Gallery]</NavItem>
```
Comment (pung-one) [inline pages/components/Navigation.js:44]: and here?

File: pages/image/[id].js [hunk 3]
```diff
@@ -160,21 +161,27 @@ const ImagePage = () => {
...
-      [id]: rangeValues,
+      [id]: rangeValuesWithTimestamp,
     };
 
     // Storing the updated range values to local storage
     localStorage.setItem("rangeValues", JSON.stringify(updatedRangeValues));
 
-    alert("Values saved!");
+    alert(
+      "Congratulations, you just made that AI feel all warm and fuzzy inside. Values saved!"
```
Comment (pung-one) [inline pages/image/[id].js:189]: hahaha

File: pages/supreme.js [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+  "Realism",
+];
+
+// Function to check if the code is running in a client-side environment and get the range values
+function getClientSideRangeValues() {
+  if (typeof window === "undefined") {
+    return null;
+  }
+  // If the window object is defined, return the stringified range values from local storage
+  return JSON.stringify(localStorage.getItem("rangeValues"));
```
Comment (pung-one) [inline pages/supreme.js:65]: very creative solution ;)

**Review by pung-one — APPROVED** [review]
looks good!

File: pages/components/Navigation.js (hunk 1 above)
Comment (CTGitWhiz) [reply pages/components/Navigation.js:41]: I want it there :D thats all...

File: pages/components/Navigation.js (hunk 2 above)
Comment (CTGitWhiz) [reply pages/components/Navigation.js:44]: I want it there :D thats all...

**Comment by CTGitWhiz** [comment]
--> last commit - Improved mobile view compatibility for Image and Tooltip components

**Pull request merged**